src/compiler/crystal/tools/doc/generator.cr

```diff
@@ -360,7 +360,13 @@ class Crystal::Doc::Generator
     filename[@base_dir.size..-1]
   end
 
-  record RelativeLocation, filename : String, line_number : Int32, url : String? do
+  class RelativeLocation
+    setter show_line_number
```

Comment:
Make this a property and remove it from the getter macro please

Reply:
Ah yes that's a good idea